src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+from django.conf import settings
+
+import httplib
+import json
+
+
+NLCD_MAPPING = {
+    11: ['water', 'Water'],
+    21: ['urban_grass', 'Urban- or Tall-Grass'],
+    22: ['li_residential', 'Low-Intensity Res.'],
+    23: ['hi_residential', 'High-Intensity Res.'],
+    24: ['industrial', 'Industrial &c.'],
```

Comment:
[Indeed it is!](http://designshack.net/articles/typography/why-i-love-ampersands-you-should-too/) Although in the interest of usability, I would recommend considering the [more common](https://en.wikipedia.org/wiki/Ampersand#Usage) representation "etc."

(The [bikeshedding](https://en.wikipedia.org/wiki/Parkinson%27s_law_of_triviality) nature of this comment is not lost on me)